Microsoft.DotNet.Try.ProjectTemplate.IntegrationTests/TutorialTemplateTests.cs

```diff
@@ -41,7 +42,18 @@ public async Task When_the_template_is_installed_verify_works()
             var baseDirectory = Create.EmptyWorkspace().Directory;
             var outputDirectory = baseDirectory.CreateSubdirectory("outputTemplate");
             await InstallTemplateAndCreateProject(baseDirectory, outputDirectory);
-
+            var dotnet = new Dotnet(outputDirectory);
+            //The template targets 3.0 hence verify should run against 3.0 and not 2.1.503 used in the solution directory
+            await dotnet.New("global.json", "--sdk-version 3.0.100-preview3-010431");
+
+            var projectFile = outputDirectory.GetFiles("*.csproj").Single();
+            
+            var result = await dotnet.Build(projectFile.FullName);
+            Console.WriteLine(String.Join(";", result.Output));
+            Console.WriteLine(String.Join(";", result.Error));
+            Console.WriteLine(String.Join(";", outputDirectory.GetFiles("*.json").Select(file => file.Read())));
+            Console.WriteLine(String.Join("\n", (await dotnet.Execute("--info")).Output));
```

Comment:
If this is for debugging purposes it may be hidden by the test runner. Logging to PocketLogger and using the `LogToPocketLoggerAttribute` will work around this.